tests/vmi_multus_test.go

```diff
@@ -870,6 +871,60 @@ var _ = Describe("SRIOV", func() {
 		It("[test_id:3957]should connect to another machine with sriov interface over IPv6", func() {
 			pingThroughSriov("fc00::1/64", "fc00::2/64")
 		})
+
+		Context("VLANed SR-IOV network", func() {
+			var (
+				vlanedVMI1   *v1.VirtualMachineInstance
+				vlanedVMI2   *v1.VirtualMachineInstance
+				nonVlanedVMI *v1.VirtualMachineInstance
+			)
+
+			const (
+				cidrVlaned1            = "192.168.0.1/24"
+				cidrVlaned2            = "192.168.0.2/24"
+				cidrNonVlaned          = "192.168.0.3/24"
+				sriovVlanNetworkName   = "sriov-vlan"
+				sriovNoVlanNetworkName = "sriov-no-vlan"
+			)
+
+			BeforeEach(func() {
+				createNetwork(sriovVlanNetworkName, tests.NamespaceTestDefault, sriovConfVlanCRD)
+				createNetwork(sriovNoVlanNetworkName, tests.NamespaceTestDefault, sriovConfNoVlanCRD)
+				vlanedVMI1 = createSriovVMI([]string{sriovVlanNetworkName})
+				// manually configure IP/link on sriov interfaces because there is
+				// no DHCP server to serve the address to the guest
+				configInterface(vlanedVMI1, "eth1", cidrVlaned1, "\\# ")
```

Comment:
why is there a space after the #? `"\\# "`

Reply:
Sorry for the late reply. 
This is done in other tests in this nodule as well. I guess it is to distinguish between "regular" `#` or `$` and a prompt.
anyway, now that we a a shiny new general [Prompt expression](https://github.com/kubevirt/kubevirt/blob/e6fa79d7220f4c5063aa2e1620d6b259bf808a85/tests/utils.go#L247)  
I made the test to use that one.